src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -1081,10 +1097,12 @@ private void model_PropertyChanged(object sender, System.ComponentModel.Property
                     RaisePropertyChanged("Width");
                     RaisePropertyChanged(nameof(ModelAreaRect));
                     UpdateAllGroupedGroups();
+                    UpdateProxyPortsPosition();
```

Comment:
check performance here...
When moving a group (drag with mouse) ...these properties can get changed a lot of times

Reply:
Ok, I will check an alternative for this.

Reply:
check if performance is an issue first...then check alternatives
No use in doing premature optimization